closer.go

```diff
@@ -39,3 +39,11 @@ func (w writeCloser) Close() error {
 	}
 	return nil
 }
+
+var _ io.WriteCloser = (*nopWriterCloser)(nil)
+
+type nopWriterCloser struct {
+	io.Writer
+}
+
+func (nopWriterCloser) Close() error { return nil }
```

Comment:
It already exists in the stdlib: https://pkg.go.dev/io#NopCloser

Reply:
Not for writers.

Reply:
https://github.com/golang/go/issues/22823